### PR TITLE
ci(eslint): use eslint+prettier with ci strictly

### PR DESCRIPTION
**Pull request by manudeli**
**What**: use eslint+prettier with ci strictly

**Why**: strict auto format with eslint+prettier will make us maintain code quality easily. After setup like this, I want to make sure my code quality with more strict eslint rule before commiting in my local machine, not after GitHub Action with loose prettier rule on pr opened

**How**: eslint+prettier setup, use it in ci.yml

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+dist
```
Comment (manudeli) [inline .eslintignore:1]: Ignore eslint rule in built source

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+  pull_request:
+    types: [opened, synchronize, reopened]
+
 jobs:
-  tests-e2e:
+  ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        command: ['pnpm lint', 'pnpm typescript', 'pnpm test']
```
Comment (manudeli) [inline .github/workflows/ci.yml:12]: All quality check concurrently in single file

**Review by manudeli — COMMENTED** [review]
@bvaughn 
I leave comments as my intention

**Comment by manudeli** [comment]
@bvaughn Could I get your any review for this Pull Request?

**Review by bvaughn — APPROVED** [review]

**Pull request merged**